sycl/doc/extensions/experimental/sycl_ext_intel_esimd/esimd_emulator.md

```diff
@@ -66,14 +66,6 @@ To compile using Intel(R) OneAPI Toolkit:
 To run under emulation through ESIMD_EMULATOR backend:
 > `$ SYCL_DEVICE_FILTER=ext_intel_esimd_emulator:gpu ./a.out`
 
-Please note that ESIMD_EMULATOR backend cannot be picked up as default
```

Comment:
Why is this paragraph removed?

Reply:
My PR is somehow dirty with changes that are not mine. Im reverting them and will push again.